README.md

```diff
@@ -1,7 +1,88 @@
+## Contributors
+
+This repository has been developed primarily by [@adrij](https://github.com/adrijh) and [@alfonjerezi](https://github.com/alfonjerezi).
+
+## Introduction
+
+This project deploys an architecture in AWS which ingest and processes streaming data with Kinesis Flink Application and writes the output to S3 in Hudi and JSON format.
 
 ## Architecture
 
 ![Alt text](images/flink-hudi.png?raw=true "Architecture")
+
+## Documentation
+
+Articles:
+ - First article: LakeHouse Flink streaming (Pending)
+
+## Requirements
+
+- You must own an AWS account and have an Access Key to be able to authenticate. You must create a profile called "practica_cloud" and put the previous credentials in it. See [here](https://docs.aws.amazon.com/cli/latest/reference/configure/) steps to create it.
+
+- Versions:
+    - Terraform = 1.1.7
+    - hashicorp/aws = 4.54.0
+    - Python = 3.8
+
+## Infrastructure deployed
+
+This code will deploy the following infraestructure inside AWS:
+- 2 Kinesis Flink Applications
+- 2 Kinesis Data Streams
+- 1 S3 bucket
+
+## Installation
+
+Follow the instructions [here](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started#:~:text=popular%20package%20managers.-,%C2%BB,Install%20Terraform,-Manual%20installation) to install terraform
+
+Follow the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to install the AWS CLI
+
+Follow the instructions [here](https://www.python.org/downloads/release/python-3816/) to install Python 3.8
+
+## bucket and DynamoDB for terraform state deployment
+
+This small infra deployment is to be able to use remote state with Terraform. See more info about remote state [here](https://developer.hashicorp.com/terraform/language/state/remote). Commands:
+
+```bash
+
+cd infra
+cd bootstraper-terraform
+terraform init
+terraform <plan/apply/destroy> -var-file=vars/bootstraper.tfvars
+
+# Example
+cd infra
+cd bootstraper-terraform
+terraform init
+terraform apply -var-file=vars/bootstraper.tfvars
+```
+
+It is important that you choose wisely the variables declared in the "bootstraper-terrafom/vars/bootstraper.tfvars" file because the bucket name is formed using these.
+
+There will be an output printed on the terminal's screen, this could be an example:
+
+```bash
+state_bucket_name = "eu-west-1-bluetab-cm-vpc-tfstate"
+```
+
+Please copy it, we will be using it in the next chapter.
+
+## Infrastructure deployment
+
+To be able to deploy the infrastructure it's necessary to fill in the variables file ("infra/vars.tfv") and the backend config for the remote state ("terraform.tf")
+
+To deploy, the following commands must be run:
+
+```bash
+terraform <plan/apply/destroy> -var-file=vars/flink-hudi.tfvars
+```
+
+We will use the value copied in the previous chapter, the state bucket name, to give the variable remote_state_bucket its value.
+
+## To take into account (Adri fill it with the jar details)
```

Comment:
Ok! I will fill this. Also things I am missing and I can do at the same time (as you already did all of this):
1. How to send events with the Locust app
2. Little reminder to click Run on the Flink App and how to check it is working properly